client/webserver/site/src/js/markets.js

```diff
@@ -952,6 +952,24 @@ export default class MarketsPage extends BasePage {
     if (this.book) this.chart.draw()
   }
 
+  /* updateTitle update the browser title based on the midgap value and the
+   * selected assets.
+   */
+  updateTitle () {
+    // gets first price value from buy or from sell, so we can show it on
+    // title.
+    const midGapValue = this.midGap()
+
+    const market = this.market
+    const [b, q] = [market.baseCfg, market.quoteCfg]
+    const firstSymbol = b.symbol.toUpperCase()
+    const secondSybol = q.symbol.toUpperCase()
+    if (midGapValue) {
+      // more than 6 numbers it gets too big for the title.
+      document.title = `${midGapValue.toFixed(5)} ${firstSymbol}/${secondSybol}`
```

Comment:
This is gonna say something like 0.004 DCR/BTC.  The rate is inversed.

Reply:
Maybe use a dash instead of a slash. When naming a market, the base symbol goes first by convention.

Reply:
I will note that Bittrex bucks this trend

![image](https://user-images.githubusercontent.com/9373513/125963303-d7d8dd56-5232-4926-a138-72eaed5fb74e.png)

probably because of how confusing it is to have it written the other way.

Still, it's true that the base asset is usually first.

Reply:
Somewhere in the Bittrex API docs, I remember reading an acknowledgement of this mistake.

Reply:
OK fair enough.  I see the url as https://bittrex.com/Market/Index?MarketName=BTC-DCR, but I don't really strive to emulate Bittrex either way.

This PR is working well for me.  However, here's what I see presently:

![image](https://user-images.githubusercontent.com/9373513/125969705-9d6fc16b-f296-48a3-8da2-3c133efc0470.png)
![image](https://user-images.githubusercontent.com/9373513/125969751-9a81cfee-bc5f-4338-8588-c7a366aca358.png)


And here's what I'd like to see:

![image](https://user-images.githubusercontent.com/9373513/125969512-9066f5cc-138e-4671-9986-5f05cb5601c0.png)
![image](https://user-images.githubusercontent.com/9373513/125969549-21c20194-976a-4cf1-bf78-fad8a08e0cb1.png)

OR even without a dash, but it's not a big deal:

![image](https://user-images.githubusercontent.com/9373513/125969364-27e5c4f5-cad9-4240-a3d6-d416be5ae3cc.png)

Reply:
I changed it and now I am showing as your last option, without dash.